cgale/linemen.html

```diff
@@ -138,6 +138,7 @@ <h1 class="page-header">Linemen art
             <div class="row">
                 <div class="col-lg-12">
                     <p>Copyright &copy; Connie Gale 2016</p>
+                      <p> Coded By AJ Donald </p>
```

Comment:
I don't know what to put here. I'm kind of out of things to say about this reoccuring line.